### PR TITLE
feat(web-app): migrate article card component for mdx

### LINKED ISSUE #539: ArticleCard MDX component

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #539 

Migrates ArticleCard component from Gatsby theme

#### Testing / reviewing
http://localhost:3000/mdx/article-card

https://gatsby-theme-carbon.vercel.app/components/ArticleCard

### CODE REVIEW

File: services/web-app/components/article-card/article-card.module.scss [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  background: transparent;
+  text-decoration: none;
+}
+
+.tile {
+  padding: spacing.$spacing-05 25% spacing.$spacing-05 spacing.$spacing-05;
+}
+
+.title {
+  @include type.type-style('body-short-02');
```
Comment (francinelucca) [inline services/web-app/components/article-card/article-card.module.scss:36]:
```suggestion
  @include type.type-style('body-compact-02');
```
should this be `body-compact-02` instead in v11?

**Review by francinelucca — DISMISSED** [review]
LGTM!

**Review by francinelucca — DISMISSED** [review]

**Review by francinelucca — CHANGES_REQUESTED** [review]
LGTM! minor change request

**Review by francinelucca — APPROVED** [review]
🎉

**Review by andreancardona — APPROVED** [review]

**Pull request merged**